content/pages/healthcare/apply.md

```diff
@@ -66,7 +66,6 @@ To update your personal, insurance, or financial information from a previous app
 ### Additional forms for caregivers, National Guard, and Reserve Servicemembers
```

Comment:
@cweber as per @akainic - let's remove reference to national guard for now and get this one finished up. thanks.